core/common/src/IModel.ts

```diff
@@ -135,6 +135,8 @@ export interface SnapshotOpenOptions extends IModelEncryptionProps, OpenDbKey {
   lazyBlockCache?: boolean;
   /** @internal */
   autoUploadBlocks?: boolean;
+  /** @internal */
+  tempFileBase?: string;

```

Comment:
Doesn't this belong on IModelDb?

Reply:
No, it is passed as the `props` argument to `DgnDb.openIModel`. You use `IModelDb.nativeDb.getTempFileBaseName` to get it's value.